graphqlapi-scm-1.rockspec

```diff
@@ -17,6 +17,7 @@ dependencies = {
     'http ~> 1',
     'errors ~> 2',
     'vshard ~> 0.1',
+    'cartridge == 2.7.4',
```

Comment:
```suggestion
    'cartridge ~> 2',
```

But if there is some specific minimal cartridge version required, let's set it (please, do a minor research)

Reply:
get_schema was added in v2.0.1-54

Reply:
Please, do it in a separate commit